years/2019/01/solutions.ts

```diff
@@ -0,0 +1,16 @@
+export const part1 = (input: string[]): number => {
+  return input.reduce((acc: number, value: string) => (acc += calcuation(parseInt(value))), 0)
```

Comment:
`.reduce` already types these function params for us so no need to do it again

```suggestion
  return input.reduce((acc, value) => (acc += calcuation(parseInt(value))), 0)
```

<img width="340" alt="image" src="https://user-images.githubusercontent.com/24367010/228901499-015a7e9c-a02c-429a-8f06-d4da49f25be0.png">

Reply:
Nice, TIL 🚀